packages/paste-core/components/button/src/styles.ts

```diff
@@ -13,6 +13,7 @@ const ResetStyles: BoxStyleProps = {
   transition: 'background-color 100ms ease-in, box-shadow 100ms ease-in, color 100ms ease-in',
   fontFamily: 'fontFamilyText',
   fontWeight: 'fontWeightSemibold',
+  textAlign: 'unset',
```

Comment:
Why is it that we want to set it to `unset` specifically, as opposed to inherit, or left?

Reply:
I'm curious what the rest of the team thinks about this, but it's currently using `unset` because it conditionally behaves like `initial` or `inherit`. You can read more in detail on the [MDN page for `unset`](https://developer.mozilla.org/en-US/docs/Web/CSS/unset).

Also, here is a sandbox that demonstrates how this works in a table. Because there is a parent with text alignment defined (e.g. the table), the button text is aligned according to that value: https://developer.mozilla.org/en-US/docs/Web/CSS/unset

Reply:
Could you link the code part of the sandbox? I only see the result!

Reply:
Opps! Yeah here it is: https://codesandbox.io/s/button-text-alignment-in-tables-r4utl

Reply:
We only want left alignded text for link style variants, right?

Reply:
Yes, only left aligned for links.